examples/mnist-inference-web/run-server.sh

```diff
@@ -0,0 +1,8 @@
+if ! command -v python3 &> /dev/null
+then
+    echo "python3 could not be found. Running server requires python3."
```

Comment:
Can we explain why the server requires python?

Reply:
Unfortunately browsers do not with WASM and other features because of some security restrictions that's why this dummy HTTP server is needed. You can view the the HTML file but any other imports will fail with this:

```
Access to script at 'file:///Users/dilshod/Projects/burn-mac/examples/mnist-inference-web/pkg/mnist_inference_web.js' from origin 'null' has been blocked by CORS policy: Cross origin requests are only supported for protocol schemes: http, data, isolated-app, chrome-extension, chrome, https, chrome-untrusted.
```

Note: I could not use rust's http server in the same crate because the crate builds with cydlib type, so I ended up with the the easiest option.

Reply:
That's a perfect explaination, I would include that in the `run-server.sh` file :)

Reply:
OK. I put the message in the script for the context.